services/121-service/src/env.ts

```diff
@@ -272,6 +272,10 @@ export const env = createEnv({
       .optional(),
     // Cooperative Bank of Oromia recommends a MAX of 10 ALPHANUMERIC CHARACTERS
     COOPERATIVE_BANK_OF_OROMIA_NARRATIVE: z.string().max(10).optional(),
+
+    // CI / Testing
+    // These variables are ONLY to be used in CI/testing environments
+    TESTING_RUN_RESTART_SERVICE_TEST: z.stringbool().default(false),
```

Comment:
Actually...

Maybe the `e2e/`-folder should have its own `env.ts`-file... As there are more "test-only"-ENV-variables, it turns out... 
And they are currently pretty 'hidden' inside all specific test-files.. (and not type-checked etc.)

This will probably be the scope for some refactoring in AB#39403...